src/components/DLViewer.vue

```diff
@@ -1,28 +1,27 @@
 <template>
   <div class="dl-viewer">
-    <MejsPlayer v-if="isVideo" :src="src" />
+    <VideoJS v-if="isVideo" :options="videoOptions" />
     <UniversalViewer
       v-else
       :iiif_manifest="iiif_manifest"
       :iiif_manifest_url="iiif_manifest_url"
       :uv_config="parseConfig"
-      :media="media"
```

Comment:
It needs this prop.

Reply:
oops, must have taken out resolving. conflicts.